apps/betterangels-backend/notes/schema.py

```diff
@@ -90,6 +92,12 @@ class Query:
 
     tasks: List[TaskType] = strawberry_django.field(extensions=[HasRetvalPerm(TaskPermissions.VIEW)])
 
+    @strawberry_django.offset_paginated(
+        OffsetPaginated[InteractionAuthorType], extensions=[HasPerm(NotePermissions.ADD)]
+    )
+    def interaction_authors(self) -> QuerySet[User]:
+        return cast(QuerySet, get_outreach_authorized_users())
```

Comment:
`get_outreach_authorized_users` returns a queryset, so no need to cast 
```suggestion
        return get_outreach_authorized_users()
```